app/transcribe/audio_transcriber.py

```diff
@@ -228,9 +228,10 @@ def convert_wav_to_16khz_format(self, file_path: str) -> str:
             file_descritor, mod_file_path = tempfile.mkstemp(suffix=".wav")
             os.close(file_descritor)
             # print(f'Convert file {file_path} to 16khz file {mod_file_path}')
+            log_file = f"{utilities.get_data_path(app_name='Transcribe')}/logs/ffmpeg.txt"
```

Comment:
Good catch that these log files were not being produced in the correct location.